Assets/scripts/NoteObject.cs

```diff
@@ -7,6 +7,7 @@ public class NoteObject : MonoBehaviour {
 	private Note _NoteData;
 	public float destroyY = -10f; // Y coordinate under which object will self destruct
 	public Vector3 velocity = new Vector3(0, -0.22f, 0);
+	public Vector3 centerOfObject = new Vector3(.5f, -.5f);

```

Comment:
You sure it isn't (0.5f, 0.5f)? It feels a bit off to me, but that could be any number of audio/video sync issues we can resolve once we add settings for a/v sync.